src/GrantType/GrantTypeBase.php

```diff
@@ -63,12 +90,48 @@ protected function getAdditionalOptions()
     }
 
     /**
-     * @inheritdoc
+     * @param bool $forceCache
+     *
+     * @return AccessToken
      */
-    public function getToken()
+    public function getToken($forceCache = false)
```

Comment:
You're not forcing a cache to be used, you're forcing it *not* to be used. So I think `$skipCache` would be clearer